### PR TITLE
Feat: Add FAQs page

### PR DESCRIPTION
**Pull request by ScottForster400**
Created and styled faq page and added FAQ questions

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (ajstrongdev) [inline faq.html:6]: Please change title to FAQs.

File: faq.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="faq.js" defer></script>
```
Comment (ajstrongdev) [inline faq.html:8]: faq.js is not committed,

**Review by ajstrongdev — CHANGES_REQUESTED** [review]
Fix issues please.

File: faq.html (hunk 1 above)
Comment (ScottForster400) [reply faq.html]: changed title also nav basr added

File: faq.html (hunk 2 above)
Comment (ScottForster400) [reply faq.html]: JS file not needed so i have deleted the line

**Pull request merged**